libs/cli/src/generators/ui/supported-ui-libraries.json

```diff
@@ -25,7 +25,7 @@
 			"@spartan-ng/ui-core": "0.0.1-alpha.319",
 			"clsx": "^1.2.1",
 			"@spartan-ng/ui-alertdialog-brain": "0.0.1-alpha.319",
-			"@spartan-ng/ui-dialog-brain": "0.0.1-alpha.319"
+			"@spartan-ng/ui-dialog-brain": "0.0.1-alpha.320"
```

Comment:
No need to change this :) It'll be automatically changed when I release the new version!